misc/RELEASE_CHECKLIST_TEMPLATE.md

```diff
@@ -0,0 +1,83 @@
+# Release Checklist
+
+## Communication
+- [ ] Create a new issue and copy&past this checklist into it (Yeah! First Step done!)
+- [ ] Talk to the team: Who should make the release?
+- [ ] Announce a "Code-Freeze". No new Pull-Request until the release is done!
+- [ ] Checkout if we have MAJOR or MINOR changes. If not we do a PATCH release.
+- [ ] The new version will be: `vX.X.Y`
+- [ ] Identify open BUGS and add them to the next PATCH milestone (optional).
+- [ ] Identify MINOR issues and add them to the next MINOR milestone (optional).
+
+## Update to the newest version
+- [ ] Update to the current version: `git checkout develop && git pull`.
+- [ ] Create a new release branch. (`git checkout -b vX.X.Y develop`)
+
+## Build & Test
+- [ ] Update the version number of the library in `package.json` (remove the "SNAPSHOT").
+- [ ] Build the library: `npm prune && rm -rf node_modules && npm install && npm run build && npm run test`
+- [ ] Open some of the examples in your browser and visually check if it works as expected! (*We need automated tests for this!*)
+
+## History
+(*THIS IS A LOT OF WORK! WE SHOULD TRY TO automate this in the future!!*)
+
+- [ ] Get all commits since the last release: ```git log `git describe --tags --abbrev=0`..HEAD --oneline > .commits.tmp```
+- [ ] Open ".commity.tmp". and remove all commit before the last release.
```

Comment:
`.commits.tmp` (also "all commit" -> "all commits"

Seems like there should be a way to get git to do this part, optimisation for later.